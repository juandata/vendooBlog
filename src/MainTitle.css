.MainTitle {
    display: flex;
    margin-right: 22px;
    width: 102px;
    height: 52px;
    font-family: Montserrat;
    font-weight: 700;
    font-size: 42px;
    color: #1b153c;
    text-decoration: none solid rgb(27, 21, 60);
}