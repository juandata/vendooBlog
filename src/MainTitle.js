import React from "react";
import "./MainTitle.css";


const MainTitle = () => <div className="MainTitle">Blog</div>;


export default MainTitle;
