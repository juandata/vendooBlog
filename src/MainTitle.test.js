import React from 'react';
import { shallow } from 'enzyme';
import MainTitle from './MainTitle';


it('MainTitle should render correctly with no props', () => {
    const component = shallow(<MainTitle/>);
    
    expect(component).toMatchSnapshot();
  });