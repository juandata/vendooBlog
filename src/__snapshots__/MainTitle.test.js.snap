// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainTitle should render correctly with no props 1`] = `
<div
  className="MainTitle"
>
  Blog
</div>
`;
